packages/web-components/src/components/va-table/va-table.tsx

```diff
@@ -1,37 +1,47 @@
-import { Component, Element, Host, Prop, State, h } from '@stencil/core';
-import { isNumeric } from '../../utils/utils';
-import ascendingIcon from '../../assets/sort-arrow-up.svg?format=text';
-import descendingIcon from '../../assets/sort-arrow-down.svg?format=text';
-import { quicksort, reverseQuicksort } from '../../utils/dom-sort';
+/* eslint-disable i18next/no-literal-string */
+import { 
+  Component,
+  Host, 
+  Element,
+  h,
+  State,
+  Prop
+} from '@stencil/core';
 
 /**
- * This component expects `<va-table-row>` elements as children.
- * Children of each row element should be `<span>` elements. Table
- * semantics will be added and numeric columns will be right aligned.
- */
-
-/**
- * @componentName Table
- * @maturityCategory use
- * @maturityLevel best_practice
+ * This is a wrapper component whose job is to marshal the cells in slotted va-table-rows 
+ * through DOM manipulation for projection into the slots of the va-table-inner component for rendering
  */
 
 @Component({
   tag: 'va-table',
-  styleUrl: 'va-table.css',
+  styleUrl: 'va-table.scss',
   shadow: true,
 })
-export class VaTable {
-  headers: HTMLElement[] = null;
-
+export class VaTableWrapper {
```

Comment:
The class name is `VaTableWrapper` in `va-table.tsx` and in `va-table-inner.txs` the class name is `VaTable`. Is that right?

Reply:
@jamigibbs - good catch - fixed!